### ISSUE TITLE
Save allowed with different adjective for existing GDM but does not update adjective

### ISSUE THREAD

**Issue by selinad** [ClinGen/clincoded#1223]
I'm sorry I didn't check this out in previous testing.

If I create a GDM with adjective 1 and then go to create a new GDM that has same GD & M, but select different adjective, it gives the pop up that the record already exists (it's good that GDM make it unique), but then allows to go to the record where it has the 1st adjective (not the 2nd).

I'm not certain if there is a relatively quick fix to this.  Not a show stopper, but also not quite right.  Could we allow them to change the adjective of the record? (it's same record, only adjective changes).  Either that or add note that if different adjective was used for same gene, disease, moi choices, will not update the adjective (better way to say that -- idea is to have a universal message as a quick "fix").  Adding to R9 for discussion.  This is related to #1104 so will put that in Needs work until we discuss.

In future, we should have way to edit adjective, is what I am thinking -- more like adding an OMIM ID to the record....

**Comment by jimmyzhen** [comment]
Hi @selinad, @wrightmw,

The alert pop-up appears to have been implemented per the request in https://github.com/ClinGen/clincoded/issues/93.

Although I agree that we should add the ability for curators to edit the adjective, it is not a relatively quick fix. We probably want to think about at which point in the curation process curators are able to edit/change the adjective and whether there are any subsequent downstream effect potentially.

I personally think the alert pop-up is not desirable to implement the adjective edit/change ability. Maybe on the Curation-Central page (e.g. in the header)?

For R9, I think adding a note in the alert pop-up (if needed indeed) will be the quickest way to address this ticket.

**Comment by wrightmw** [comment]
@selinad @jimmyzhen 
I totally agree that the adjective should be an editable field in future, however I also agree that this should not be a blocker in R9. 
Adding a note seems the best action for now.

**Comment by jimmyzhen** [comment]
@selinad, @wrightmw,

Would either one of you be able to draft up the appropriate note text for me to add to the alert pop-up? Thanks!

**Comment by selinad** [comment]
yes - I will.  Sorry, I was part way through that then apparently distracted myself and accidentally closed the tab (sigh)

**Comment by selinad** [comment]
Hi @jimmyzhen just noticed if no adjective has been added to a GDM, can't add one to the same GDM combo...that seems somewhat problematic -- would that take a fair amount of work also?

**Comment by jimmyzhen** [comment]
@selinad, what do you mean by the "GDM combo"?

Are you saying that, if no adjective was added to a given GDM when it was created, the curators can't add one it later (after it was created)?

**Comment by selinad** [comment]
correct...let me try again, though.

**Comment by selinad** [comment]
Here we go:
![screen shot 2017-01-17 at 2 07 19 pm](https://cloud.githubusercontent.com/assets/11320314/22042055/5ea7284e-dcbe-11e6-8e9d-b20bf016cb16.png)

![screen shot 2017-01-17 at 2 07 26 pm](https://cloud.githubusercontent.com/assets/11320314/22042059/632ed8c6-dcbe-11e6-8727-4d0aa8837676.png)

![screen shot 2017-01-17 at 2 07 34 pm](https://cloud.githubusercontent.com/assets/11320314/22042063/68e47302-dcbe-11e6-96e9-97735603bf85.png)

**Comment by jimmyzhen** [comment]
Hi @selinad,

If I am not mistaken, I think your screen captures are confirming the problem you were describing in this ticket's description.

Did you get a chance to see my comment (https://github.com/ClinGen/clincoded/issues/1223#issuecomment-273280618) above?

**Comment by selinad** [comment]
Hi @jimmyzhen it's perhaps a different symptom of the same problem.  I was describing the inability to edit an adjective -- now I'm describing the inability to add one when there wasn't one before.  Perhaps those are both the result of the same code.

**Comment by selinad** [comment]
Since the adjective is set in stone when they create the GDM, it seems we should add note under the pull-downs on Create GDM page.  Should I create a separate ticket and move this ticket to R10?

**Comment by jimmyzhen** [comment]
Hi @selinad,

I think adding a note underneath the pull-down menus on the Create-GDM page can be done via this ticket, while adding the capability to add/update/delete an adjective for an existing GDM should be in a separate ticket for R10.

**Comment by selinad** [comment]
Thanks, @jimmyzhen !

How about this note (maybe in the yellow warning box style)?  @wrightmw plz feel free to edit:

"Currently, once selected, the above options cannot be altered for a gene:disease record (gene, disease, mode of inheritance, or adjective).  This includes adding an adjective to a gene:disease:mode of inheritance record that has already been created or editing an adjective associated with a record."

**Comment by selinad** [comment]
Small update:

"Currently, once selected, the above options cannot be altered for a gene:disease record (gene, disease, mode of inheritance, or adjective) once the record has been created. This includes adding an adjective to a gene:disease:mode of inheritance record that has already been created or editing an adjective associated with a record."

**Comment by jimmyzhen** [comment]
Hi @selinad,

Will this work?
![image](https://cloud.githubusercontent.com/assets/6956310/22043729/de3101a0-dcc5-11e6-9e46-cb1fafe378a3.png)

**Comment by selinad** [comment]
Looks great, except for a few edits to my text (my apologies):

"Currently, the above options (gene, disease, mode of inheritance, or adjective) cannot be altered for a Gene:Disease record  once the record has been created. This includes adding an adjective to a Gene:Disease:Mode of inheritance record that has already been created or editing an adjective associated with a record."

**Comment by jimmyzhen** [comment]
Hi @selinad,

The note has been updated:
![image](https://cloud.githubusercontent.com/assets/6956310/22043917/cba15160-dcc6-11e6-98fe-97d5f8affdb7.png)

**Comment by selinad** [comment]
Looks great -- thanks @jimmyzhen .

**Comment by wrightmw** [comment]
The note looks great... thanks

**Comment by selinad** [comment]
Yup -- note is perfect.  The issue of being able to edit adjective will be addressed in #1234 for a future release.  thx @jimmyzhen !

**Comment by kilodalton** [comment]
Included in last release (R9). Nice job and thanks for your hard work.
